database/migrations/2015_03_23_002918_create_issues_table.php

```diff
@@ -14,7 +14,7 @@ public function up() {
 		Schema::create('issues', function (Blueprint $table) {
 			$table->increments('id');
 			$table->string('title');
-			$table->text('description');
+			$table->string('description')->nullable();
```

Comment:
Personally, I think this isn't the best way to address the issue. With this change, an issue's description is limited to 255 characters (the default string length).

I think a more backwards-compatible change would be to simply have a default empty value on the column:

    $table->text('description')->default('');